crtauth/ssh.py

```diff
@@ -52,7 +56,7 @@ def base64url_decode(s):
     except:
         _, e, tb = sys.exc_info()
         raise exceptions.InvalidInputException(
```

Comment:
IIRC, that's the old way of re-raising an exception while keeping its traceback. That, or it was to keep the traceback from the old exception while raising another error or something.